controllers/discovery/initDiscoveryUserData.js

```diff
@@ -69,11 +88,21 @@ const InitDiscoveryUserData = async (req, res) => {
     });
     let suggestedUsers = usersWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
+    totalData = totalData[0]?.total_data;
+
     return res.status(200).json({
       success: true,
       message: `Fetch discovery data success`,
       suggestedUsers,
-      nextPage: page + 1
+      nextPage: page + 1,
+      total_page: Math.ceil(totalData / limit)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The response object now includes `nextPage` and `total_page`. While this is a good practice for paginated APIs, it would be better to follow a consistent naming convention. Consider renaming `total_page` to `totalPages` to match the camelCase format of `nextPage`.

```diff
- total_page: Math.ceil(totalData / limit)
+ totalPages: Math.ceil(totalData / limit)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->